src/main/resources/application.conf

```diff
@@ -76,6 +76,10 @@ backend {
 google {
   // Used by cromwell to make calls to JES and to perform GCS reads/writes.
   // The latter can be overriden if the "User Mode" is used (see below).
+  // Use "application-default" to use the default service account credentials.
+  // This is useful if you are running on a GCE VM and if you don't need
+  // user level access. See https://developers.google.com/identity/protocols/application-default-credentials
+  // for more info.
```

Comment:
You mentioned that there's something we have to do to use this on our desktops... I don't believe that's documented in the link you provided, is that worth adding here?